### PR TITLE
As a developer, I want to know exactly our plan for building and hosting the ETL and scoring code and infrastructure, so that I can plan my work.

### LINKED ISSUE #104: As a developer, I want to know exactly our plan for building and hosting the ETL and scoring code and infrastructure, so that I can plan my work.
**Description**
We need a very clear architecture diagram, with clear plans for infrastructure, for the whole backend processes of ETL, scoring, APIs, etc. 

**Solution**
A clear and concise description of the planned process for building the backend, including questions such as:

* The exact infrastructure (e.g., RDS with PostGIS, EC2 for running ETL scripts) with level of access needed  
* Code hosting (which repos)
* Build server
* How front-end accesses data

**Tasks**
* Build architecture diagram with clear descriptions

**Definition of "Done"**
* We feel like every question we need to answer about the where/what of the plan is answered and documented, so that we can start building with confidence.

### PR DESCRIPTION
**Pull request by switzersc-usds**
Add diagrams detailing our geo data pipeline:
* Detailed flow for data inputs, census data, score creation, and geo
data generation (geoJSON and MVT)
* Include diagram showing vision for geoplatform ownership

Closes #104

### CODE REVIEW

**Review by NatHillardUSDS — APPROVED** [review]
LGTM!

File: docs/architecture/geodata-pipeline-arch-geoplatform.mmd [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    Census["Census TIGER Data"]-->CGTiger
+    subgraph "Owned by Geoplatform"
+        CGTiger["Create GeoJSON from Shapefile with osgeo/gdal"]-->TS3
+        
+    end
+    TS3("TIGER GeoJSON (S3)")-->CGJ
+    Score["Create Score CSV"]--Event Notification-->CSV
+    CSV("CSV (S3)")--"Event Notification (Geoplatform)"-->CGJ
+    subgraph "Owned by Geoplatform"
+        CGJ["Combine (ogr2ogr)+ Create GeoJSON"]--Event Notification-->GeoJSON
```
Comment (lucasmbrown-usds) [inline docs/architecture/geodata-pipeline-arch-geoplatform.mmd:13]: @switzersc-usds -- is this part all automated? Or do they need to manually run some of this?

**Comment by ginabeena** [comment]
I wish Github rendered SVGs in page. If these are the same diagrams as the mermaid ones I saw that are vector, it LGTM :shipit:

File: docs/architecture/geodata-pipeline-arch-geoplatform.mmd (hunk 1 above)
Comment (switzersc-usds) [reply docs/architecture/geodata-pipeline-arch-geoplatform.mmd:13]: Automated!

**Pull request merged**